src/main/java/uk/gov/ons/census/casesvc/service/RmCaseUpdatedService.java

```diff
@@ -224,4 +232,8 @@ private void validateCase(Case caze) {
       throw new RuntimeException("Case missing mandatory fields after RM Case Updated");
     }
   }
+
+  public boolean isOaPresent(Case originalCase) {
+    return originalCase.getOa() != null;
+  }
```

Comment:
This not used?

Reply:
Good spot.  That got left in by mistake.  Removed.